1.10/cli/command-reference/dcos-config/index.md

```diff
@@ -4,17 +4,23 @@ menu_order: 2
 ---
 
 # Description
-This command manages the DC/OS configuration file. A default configuration file is created during initial DCOS CLI setup and is located in `~/.dcos/dcos.toml`.
+This command manages the DC/OS configuration file created when you run [dcos cluster setup](/docs/1.10/cli/command-reference/dcos-cluster/dcos-cluster-setup). The configuration file is located in `~/.dcos/clusters/<cluster_id>/dcos.toml`. If you have not changed any configuration properties you should see this output when you run `dcos config show`:
+
+    cluster.name <cluster_name>
+    core.dcos_acs_token ********
+    core.dcos_url <cluster_url>
+    core.ssl_verify `true` or `false`
+
 
 ## Environment variables
-Configuration properties all have corresponding environment variables. If a property is in the "core" section (ex. "core.foo", it corresponds to environment variable DCOS_FOO. All other properties (ex "foo.bar") correspond to environment variable DCOS_FOO_BAR.
+Configuration properties have corresponding environment variables. If a property is in the "core" section (ex. "core.foo", it corresponds to environment variable DCOS_FOO. All other properties (ex "foo.bar") correspond to environment variable DCOS_FOO_BAR.
```

Comment:
Suggest: If a property is in the `core` section (e.g., `core.foo`), it corresponds to the environment variable `DCOS_FOO`. All other properties (e.g., `foo.bar`) correspond to the environment variable `DCOS_FOO_BAR`.